### DISCUSSION TITLE
Import profiles or IDEX Info

### DISCUSSION THREAD

**Discussion by Fisheye3D** [OrcaSlicer/OrcaSlicer#1311] [General]
I was wondering if the ability to pull profiles from PS to orca will be or is implemented, I have a craftbot flow idex xl I would love to use with Orca but no way to create cfg for itn in Orca

**Comment by fr3ak2402** [comment]
I will create a profile for you for the craftbot flow idex xl.

Can you please send me your ini?

I am currently working on a converter for the profiles...

**Reply by Fisheye3D** [reply]
[CraftbotPS.ini.zip](https://github.com/SoftFever/OrcaSlicer/files/11710759/CraftbotPS.ini.zip)

Because of the jacked up way Craftbot made there firmware, it needs separate profiles for each mode as seen in SS, Not sure if your knowledge can combine them but each one has its own bed size modified. 
![image](https://github.com/SoftFever/OrcaSlicer/assets/78997080/b95d4ec7-d32c-4340-8d1d-a64d97ce2a90)

**Reply by fr3ak2402** [reply]
Don't worry, I'll manage. I have created most of the printer profiles for the OrcaSlicer. The setup is not a problem. You will also have all separate machines like in the picture.

I will finish the profile tomorrow morning

**Reply by Fisheye3D** [reply]
Am Curious if it is even possible being It needs to have Gcode substitution and I dont see that in Orca. 
![image](https://github.com/SoftFever/OrcaSlicer/assets/78997080/eb449ba1-be4a-41d4-824d-db4c82fd9072)

**Reply by Ukrevealed** [reply]
could i please get a copy of these profiles as i have the craftbot flow idex and world be great to be able to use them on orca. many thanks

**Reply by Fisheye3D** [reply]
Haven't heard anything since about them in Ocra.

**Comment by Fisheye3D** [comment]
@fr3ak2402 Here is a updated Craftbot.ini credit goes to [polishx3](https://github.com/polishx3/Craftbot-Prusaslicer/commits?author=polishx3) 
[Craftbot-Prusaslicer-main.zip](https://github.com/SoftFever/OrcaSlicer/files/11736870/Craftbot-Prusaslicer-main.zip)

**Reply by fr3ak2402** [reply]
Okay, thank you. Then I'll revise the profile again...

**Reply by Fisheye3D** [reply]
Thank you, Do you post them anywhere specific?

**Reply by Miraque81** [reply]
Hi there, is there a profile available for the craftbot plus? I don’t like the craftware slicer and it would be nice to use the orcaslicer

**Comment by Fisheye3D** [comment]
Just wondering if any further info on this?

**Comment by sunshinedrone** [comment]
so we can't import SS profile to Orca ?

**Comment by guzior-lps** [comment]
I'll ask here to not write new post:
Is there and option to somehow make printer with 1 hotend and 2 extruders (im using Phaetus TaiChi) or regular IDEX printer (with 2 color printing)?
Next question:
Is there an option to remove system presets when i'll build my own ?

**Comment by PuszInPusz** [comment]
Hey, any news on this one? I also have a Craftbot Flow Idex and would like to get away from Craftware (it sucks in my opinion).

**Comment by cryoglobe** [comment]
Hello fr3ak2402
Any info or more infos about how to import Prusa Profiles into Orca ?   
It's really the starting point for me to try Orca , based on my current Prusa Slicer profiles ( to be able to compare ) 
I have Sovol S04 ( idex ) and Zonestar Z9V5Pro ( both mixing and switching hotends )

**Comment by awabom** [comment]
I have published my SV04 profiles for OrcaSlicer on my github: https://github.com/awabom/3dprint/tree/main/OrcaSlicer-SV04

They go in your user configuration folder, so they're not selectable in the add-printer wizard.

EDIT: And, they require postprocessing-scripts I made for PrusaSlicer but also works for OrcaSlicer (also on my github)

**Reply by cryoglobe** [reply]
Thanks a lot ... this will really help me a lot to start with Orca :)    I'll dig your github for all other valuable informations ;)

**Reply by StevenHU0081212333** [reply]
How can add the files to Ocra，my email info@innocube3d.com

**Comment by Miraque81** [comment]
Is a craftbot plus profile available? Thx

awabom ***@***.***> schrieb am So. 5. Nov. 2023 um 16:31:

**Comment by ScoRpioN-18** [comment]
i'm looking also for craftbot plus profile. I made one, but I cannot add acceleration and jerk, so no printing estimating time, but the worst part is the over-extrude at the corners. I cannot calibrate with PA, all the lines are the same. 
Tried all the  g code flavor , but ads M codes that CB+ not supported
![sample](https://github.com/SoftFever/OrcaSlicer/assets/162314967/183c1605-ab03-4d31-a993-beb31b9dff57)

Any ideas?
 Thanks!

**Comment by ScoRpioN-18** [comment]
[CraftbotProPlus.zip](https://github.com/SoftFever/OrcaSlicer/files/14800638/CraftbotProPlus.zip)
I finally made one with good results for PLA and TPU.
I found how to calibrate the linear pressure advance (M900).
M900 K7 for Sunlu PLA and M900 K40 for Polymaker TPUHF
